WordPress/build.gradle

```diff
@@ -501,9 +500,23 @@ task violationCommentsToGitHub(type: se.bjurr.violations.comments.github.plugin.
     ]
 }
 
-task printVersionName {
+task "updateVersionProperties" {
     doLast {
-        println android.productFlavors.vanilla.versionName
+        ant.propertyfile(
+                file: "../version.properties") {
+            entry( key: "$key", value: "$value")
+        }
+    }
+}
+
+task "printVersionName" {
+    doLast {
+        def key = "$productKey" + "."
```

Comment:
I don't think this makes it very clear that `productKey` is supposed to be passed as a project property.

Reply:
Tracking in https://github.com/wordpress-mobile/WordPress-Android/issues/14886